plugins/CircuitExplorer/plugin/CircuitExplorerPlugin.cpp

```diff
@@ -405,6 +414,14 @@ void CircuitExplorerPlugin::init()
             "make-movie",
             [&](const MakeMovieParameters& params) { _makeMovie(params); });
 
+        PLUGIN_INFO << "Registering 'trace-aterograde' endpoint" << std::endl;
```

Comment:
Small typo here: __trace-anterograde__ instead of __trace-aterograde__
`PLUGIN_INFO << "Registering 'trace-anterograde' endpoint" << std::endl;`

Reply:
Fixed